tests/utils.go

```diff
@@ -94,18 +94,6 @@ const (
 	DiskCustomHostPath     = "disk-custom-host-path"
 )
 
-func CreateSecret(name, namespace string, data map[string]string) {
-	virtCli := kubevirt.Client()
-
-	_, err := virtCli.CoreV1().Secrets(namespace).Create(context.Background(), &k8sv1.Secret{
-		ObjectMeta: metav1.ObjectMeta{Name: name},
-		StringData: data,
-	}, metav1.CreateOptions{})
-	if !errors.IsAlreadyExists(err) {
```

Comment:
If the secret already existed, no error would have been raised.
But now you have changed it per what I can see.

Please reason for it.

Reply:
The secret should be created once, the fact that it go slilently if it already exist is prune to error and unexpected behavior.

The global cleanup test suite delete only secrets that has `kubevirt.io/secret` label, and there are places the secret created without it. 
I posted PR to address how  secret are created in tests https://github.com/kubevirt/kubevirt/pull/11547
Also, I plan to post another PR to make the tests cleanup to not filter secrets by the mentioned label and just delete them all (those in Kubevirt tests namespace).

My guess, it ignores the error to not fail on secrets that were created by the previous test and did not  cleaned up by the global test suite cleanup (due to missing label).

Reply:
I thought we can do it in this PR because with this changes secrets will be created with the `kubevirt.io/secret` label and will get deleted by the global test suite cleanup, unlike the current state where secrets could remain.

I changed it to follow the current state - tolerate already-exist errors.